crates/paimon/src/spec/schema.rs

```diff
@@ -51,3 +51,176 @@ pub struct DataField {
     typ: DataType,
     description: Option<String>,
 }
+
+impl DataField {
+    pub fn new(id: i32, name: String, typ: DataType, description: Option<String>) -> Self {
```

Comment:
The `description` is optional; perhaps we could introduce an API named `with_description` to make `new()` short.

Reply:
Ok, I'll edit this.